v1/plugin/stream_collector_proxy.go

```diff
@@ -94,14 +138,22 @@ func streamRecv(
 	}
 }
 
-func (c *StreamProxy) SetConfig(context.Context, *rpc.ConfigMap) (*rpc.ErrReply, error) {
+func (p *StreamProxy) setMaxCollectDuration(d time.Duration) {
+	p.maxCollectDuration = d
+}
+
+func (p *StreamProxy) setMaxMetricsBuffer(i int64) {
+	p.maxMetricsBuffer = i
+}
+
+func (p *StreamProxy) SetConfig(context.Context, *rpc.ConfigMap) (*rpc.ErrReply, error) {
 	return nil, nil
 }
```

Comment:
I assume yes. It only looks like added by me, but it exists in master [v1/plugin/stream_collector_proxy.go#L97](https://github.com/intelsdi-x/snap-plugin-lib-go/blob/master/v1/plugin/stream_collector_proxy.go#L97). 

I think that a good point is to move `setMaxCollectDuration` and `setMaxMetricsBuffer` (unexported) to the end of this file - I will do that.